packages/flutter/test/cupertino/button_test.dart

```diff
@@ -27,8 +27,8 @@ void main() {
     final RenderBox buttonBox = tester.renderObject(find.byType(CupertinoButton));
     expect(
       buttonBox.size,
-      // 1 10px character + 16px * 2 is smaller than the default 44px minimum.
-      const Size.square(44.0),
+      // 1 10px character + 20px * 2 = 50.0
+      const Size(50.0, 44.0),
```

Comment:
Is this going up because of the padding from `kCupertinoButtonPadding` for the large sized style?